physionet-django/notification/templates/notification/email/notify_owner_data_access_review_withdrawal.html

```diff
@@ -0,0 +1,7 @@
+Dear {{ owner.get_full_name }},
+
+This is to inform you, that the user {{ user.get_full_name }} wished to stop reviewing data access requests for the project {{ project }}.
```

Comment:
Minor suggestions: (1) no need for the comma in `inform you, that` (2) instead of `wished to`, `has decided to` sounds more natural.